### PR TITLE
Implement basic ui for MainActivity

### PR DESCRIPTION
**Pull request by bkaryna**
## Basic ui/ux sketch so that we could move forward with developing app functionality. 

image

### CODE REVIEW

File: app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+import com.google.android.material.chip.Chip;
+import com.google.android.material.chip.ChipDrawable;
+import com.google.android.material.chip.ChipGroup;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.List;
+
+public class CreatePartyActivity extends AppCompatActivity implements View.OnClickListener {
+    private int year, month, day, hour, minute;
```
Comment (ixrk) [inline app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java:32]: Dates are hard. Try to store them in a high level class, such as `LocalDateTime` or `LocalDate` with `LocalTime`. [java.time](https://developer.android.com/reference/java/time/package-summary)

File: app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+                    return true;
+                }
+                return false;
+            }
+        });
+    }
+
+    //onclick listeners for date/time pickers, food/... chip input
+    @Override
+    public void onClick(View v) {
```
Comment (ixrk) [inline app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java:170]: I think we should use if/else here.

File: app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.List;
+
+public class CreatePartyActivity extends AppCompatActivity implements View.OnClickListener {
+    private int year, month, day, hour, minute;
+    private ActivityCreatePartyBinding viewBinding;
+    Switch dressCode = null;
+    private List<String> tagListFood = Arrays.asList("Polish", "Pizza", "Kebab", "Sushi", "Asian", "Italian", "Burgers", "Mexican", "Vietnamese"); //make it final?
+    private List<String> tagListDrinks = Arrays.asList("Coke", "Sprite", "Fanta", "Beer", "Warka", "Heineken", "Vodka", "Whiskey", "Martini", "Shots");
```
Comment (ixrk) [inline app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java:36]: Don't include trademarks. Use categories like "light beer", "craft beer", "cocktails", "spirits", "soft drinks", "juices", "tea", "coffee" etc. Be neither too specific nor too broad.

**Review by ixrk — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/github/partymakers/partymaker/MainActivity.java [hunk 4]
```diff
@@ -36,7 +42,8 @@ protected void onCreate(Bundle savedInstanceState) {
         user = firebaseAuth.getCurrentUser();
         if (user == null) {
             createLoginIntent();
-        }
+        } else
+            setUserInfoTextViews(viewBinding, user);
```
[the remark this answers is not in the document]
Comment (bkaryna) [reply app/src/main/java/com/github/partymakers/partymaker/MainActivity.java:46]: Done ✅

File: app/src/main/java/com/github/partymakers/partymaker/MainActivity.java [hunk 5]
```diff
@@ -25,6 +26,11 @@
     private FirebaseUser user;
     private ActivityMainBinding viewBinding;
 
+    protected void setUserInfoTextViews(ActivityMainBinding view, FirebaseUser usr) {
```
[the remark this answers is not in the document]
Comment (bkaryna) [reply app/src/main/java/com/github/partymakers/partymaker/MainActivity.java:29]: Done ✅

File: app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java (hunk 2 above)
Comment (bkaryna) [reply app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java:170]: Done ✅

File: app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java (hunk 3 above)
Comment (bkaryna) [reply app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java:36]: Done ✅

File: app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java (hunk 1 above)
Comment (bkaryna) [reply app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java:32]: DIY 👀

**Review by ixrk — APPROVED** [review]

**Pull request merged**